components/storage/blockdevice/COMPONENT_SD/SDBlockDevice.cpp

```diff
@@ -872,7 +872,9 @@ uint32_t SDBlockDevice::_go_idle_state()
         if (R1_IDLE_STATE == response) {
             break;
         }
+#if defined(MBED_CONF_RTOS_PRESENT)
```

Comment:
Please see my [comment](https://github.com/ARMmbed/mbed-os/pull/11721/files/b1731ce3c826eaf385549ed08e068a21e56148a0#r343506548) about this subject.